README.md

```diff
@@ -127,6 +127,24 @@ Download specific tests based on their id on the Rainforest dashboard
 rainforest download 33445 11232 1337
 ```
 
+#### Running Local RFML Tests Only
+
+If you want to run a local set of RFML files (for instance in a CI environment), use the `run -f` option:
+
+```
+rainforest run -f [FILES OR FOLDERS]
+```
+
+`run -f` accepts any number of files and folders as arguments (folders will be recursively checked for `*.rfml` files). All embedded tests must be included within `FILES OR FOLDERS`.
```

Comment:
Do you think it's clear enough that we accept pattern matched paths?

Reply:
We don't accept pattern matched paths (the shell does expansion). I think "power users" will assume that.